crates/sui-json-rpc/src/coin_api.rs

```diff
@@ -801,8 +801,10 @@ mod tests {
             assert!(response.is_err());
             let error_result = response.unwrap_err();
             let error_object: ErrorObjectOwned = error_result.into();
-            let expected = expect!["-32000"];
```

Comment:
So the reason why I do this is so that the expected! crate can automatically update snapshots for me

Reply:
This is new to me, which snapshot?

Reply:
The snapshot is in the code

https://docs.rs/expect-test/latest/expect_test/

Anywhere there's expected, it'll automatically update if expected vs actual changes, if the person running the test allows so